routers/repo/release.go

```diff
@@ -67,7 +67,13 @@ func Releases(ctx *context.Context) {
 
 	releases, err := models.GetReleasesByRepoID(ctx.Repo.Repository.ID, page, limit)
```

Comment:
We should also make sure that if we're not the owner, then this doesn't return drafts. I know that we check in the for loop below, but right now it's possible that we end up displaying fewer than `limit` releases on the first page, even if there are more than `limit` displayable releases in total.